docker-compose.db.yml

```diff
@@ -10,9 +10,14 @@ services:
       POSTGRES_DB: ${API_DB_DATABASE}
     ports:
       - ${API_DB_PORT}:5432
+    volumes:
+      - pg-data:/var/lib/postgresql/data
```

Comment:
Persist database in docker volume (no need to migrate everytime when there are no database changes)

Reply:
Nice, thanks!

Reply:
We can host DB in a separate RDS instance instead of sharing t2.micro resources between API and DB :bulb:

Reply:
Yes, I agree @sjedlikowski but how does it affect the docker-compose file with db configuration for local development? 

I guess that for production usage we will indeed use postgres provided by Heroku or RDS.

Reply:
Aaaargh, I'm getting blind :-|
I thought that it was in the `Dockerfile`...